cloud/jenkins/pgo-v2-operator-aws-openshift-4.yml

```diff
@@ -0,0 +1,14 @@
+- job:
+    name: pgo-v2-operator-aws-openshift-4
```

Comment:
All other v2 jobs are called without this addition, so let's keep it consistent.